src/components/Sidebar/index.jsx

```diff
@@ -1,7 +1,19 @@
 import "./styles.css";
 
 const Sidebar = () => {
-  return <p className="sidebar">Sidebar</p>;
+  return (
+    <ul className="sidebar">
```

Comment:
good semantic. would be better if we use nav as well.

Reply:
Yesssss.you are right
Thanks